### PR TITLE
added volume size  command to glustercli

### PR DESCRIPTION
**Pull request by Madhu-1**
Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@prashanthpai please review

**Comment by aravindavk** [comment]
Can we accommodate these details in Volume Status output itself?

**Comment by prashanthpai** [comment]
> Can we accommodate these details in Volume Status output itself?

The size API is an expensive operation as it involves (fuse) mounting a volume. Is that acceptable ?

**Comment by aravindavk** [comment]
> The size API is an expensive operation as it involves (fuse) mounting a volume. Is that acceptable ?

I thought Volume status command is also calling `client.VolumeStatus`

**Comment by Madhu-1** [comment]
@aravindavk  volume status it calls BricksStatus

**Comment by prashanthpai** [comment]
> @aravindavk volume status in CLI calls BricksStatus api

It's been named so in CLI to preserve backward compatibility

File: glustercli/cmd/volume.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ const (
 	helpVolumeInfoCmd   = "Get Gluster Volume Info"
 	helpVolumeListCmd   = "List all Gluster Volumes"
 	helpVolumeStatusCmd = "Get Gluster Volume Status"
+	helpVolumeSizeCmd   = "Get Gluster Volume Size"
```
Comment (prashanthpai) [inline glustercli/cmd/volume.go]: `Get Gluster Volume Size Usage`

File: glustercli/cmd/volume.go [hunk 2]
```diff
@@ -432,6 +434,28 @@ var volumeStatusCmd = &cobra.Command{
...
+		vol, err := client.VolumeStatus(volname)
+		if err != nil {
+			if verbose {
+				log.WithFields(log.Fields{
+					"error": err.Error(),
+				}).Error("error getting volume size")
+			}
+			failure("Error getting Volume size", err, 1)
+		}
+		fmt.Println("Volume :", volname)
```
Comment (prashanthpai) [inline glustercli/cmd/volume.go]: You can remove the extra space before `:` but retain the one after it.

Here and below.

File: glustercli/cmd/volume.go (hunk 1 above)
Comment (Madhu-1) [reply glustercli/cmd/volume.go]: done

File: glustercli/cmd/volume.go (hunk 2 above)
Comment (Madhu-1) [reply glustercli/cmd/volume.go]: done

**Comment by prashanthpai** [comment]
retest this please

**Pull request merged**